src/fides/api/models/privacy_experience.py

```diff
@@ -155,64 +167,36 @@ class ExperienceTranslationBase:
     title = Column(String)
 
 
-class DeprecatedConfigFields:
-    """Experience Config fields that are pending removal
-
-    # TODO remove all fields from PrivacyExperienceConfig model.
-    # Some are moving to ExperienceTranslation, others are being removed entirely.
-    """
-
-    accept_button_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    acknowledge_button_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    banner_description = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    banner_enabled = Column(String(), index=True)  # TODO pending removal
-    banner_title = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    description = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    is_default = Column(Boolean, nullable=False, default=False)  # TODO will be removed
-    privacy_policy_link_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    privacy_policy_url = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    privacy_preferences_link_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    reject_button_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    save_button_label = Column(
-        String
-    )  # TODO pending removal in favor of ExperienceTranslation
-    title = Column(String)  # TODO pending removal in favor of ExperienceTranslation
-    version = Column(Float, nullable=False, default=1.0)  # TODO pending removal
-
-
-class PrivacyExperienceConfig(
-    DeprecatedConfigFields, PrivacyExperienceConfigBase, Base
-):
+class PrivacyExperienceConfig(PrivacyExperienceConfigBase, Base):
     """
     The Privacy Experience Configuration model that stores shared configuration for Privacy Experiences.
 
     - Translations, Notices, and Regions (via Privacy Experiences) are linked to this resource.
     """
 
+    allow_language_selection = Column(
```

Comment:
nit: looks like these overrides are now more common than the base values, if i'm understanding correctly. so you _could_ switch around to make these the base values and instead override them in `PrivacyExperienceConfigHistory` and it'd be a bit cleaner? but obviously not a big deal...

Reply:
That was in the original draft! but it failed mypy - similar to how we have to have the fields optional on the base Privacy Notice pydantic schemas and override them to to make required on classes that inherit, it was that same pattern that was failing